near-sdk-macros/src/lib.rs

```diff
@@ -81,13 +81,41 @@ use syn::{parse_quote, ItemEnum, ItemImpl, ItemStruct, ItemTrait, WhereClause};
 ///
 /// }
 /// ```
+///
+/// Contract Source Metadata Standard:
+///
+/// By using `contract_metadata` as an argument `near_bindgen` will populate the contract metadata
+/// according to NEP-330 standard. This still applies even when `#[near_bindgen]` is used without
```

Comment:
Make NEP-330 a link to the document, so it makes readers aware on where to find it.

Reply:
@shariffdev Please, don't forget about this one

Reply:
Done